pylint/checkers/base_checker.py

```diff
@@ -162,7 +162,10 @@ def check_consistency(self):
             existing_ids.append(message.msgid)
 
     def create_message_definition_from_tuple(self, msgid, msg_tuple):
-        if implements(self, (IRawChecker, ITokenChecker)):
+        if isinstance(self, (BaseTokenChecker, BaseRawFileChecker)):
+            default_scope = WarningScope.LINE
+        # TODO: Interfaces: Deprecate looking for implements here # pylint: disable=fixme
+        elif implements(self, (IRawChecker, ITokenChecker)):
             default_scope = WarningScope.LINE
         else:
             default_scope = WarningScope.NODE
```

Comment:
```suggestion
        warn_msg = f"'{type(self)}' use an old rejected PEP from 2001 and should inherit from {} instead, it will be deprecated in 3.0 yadayada"
        if isinstance(self, (BaseTokenChecker, BaseRawFileChecker)):
            default_scope = WarningScope.LINE
        elif implements(self, (ITokenChecker)):
            warnings.warn(warn_msg.format("BaseTokenChecker")
            default_scope = WarningScope.LINE
        elif implements(self, (ITokenChecker)):
              warnings.warn(warn_msg.format("BaseRawFileChecker")
            default_scope = WarningScope.LINE           
```

Or are you thinking of something more complicated than this ?

Reply:
No, but I wanted to fix our own code first before deprecating. Currently we do not fully support not using ``__implements__`` as we don't check for `IAstroidChecker`, so deprecating without having actual support for the new thing seems premature.